src/java.desktop/windows/classes/sun/awt/windows/WDataTransferer.java

```diff
@@ -275,7 +266,8 @@ public Object translateBytes(byte[] bytes, DataFlavor flavor, long format,
             {
                 try {
                     charset = new String((byte[])localeTransferable.
-                        getTransferData(javaTextEncodingFlavor), "UTF-8");
+                        getTransferData(javaTextEncodingFlavor),
+                                         StandardCharsets.UTF_8);
```

Comment:
```suggestion
                        getTransferData(javaTextEncodingFlavor),
                                        StandardCharsets.UTF_8);
```
The parameter on the second line should probably be aligned with the first parameter as it's done in the snippet above.

Reply:
it is aligned already, the StandardCharsets.UTF_8 is parameter of "new String()", not the getTransferData.

Reply:
Ah, right!
But it's confusing: it looks as if `StandardCharsets.UTF_8` is a parameter to `getTransferData`. Maybe avoid breaking the line and leave `UTF_8`  on the same line? If you import `UTF_8` and `UTF_16LE` statically, line break is unnecessary.

Reply:
I am fine to do that, if there are no objections I can change the whole fix.

Reply:
> 
> I am fine to do that, if there are no objections I can change the whole fix.

Modifying the entire changeset seems like an overkill.
Using static imports in only one file is _inconsistent_, yet it makes the places where the encodings are used clearer.

Reply:
Updated